dev/com.ibm.websphere.appserver.features/visibility/public/jaxws-2.3/resources/l10n/com.ibm.websphere.appserver.jaxws-2.3_ja.properties

```diff
@@ -0,0 +1,19 @@
+###############################################################################
+# Copyright (c) 2012 IBM Corporation and others.
+# All rights reserved. This program and the accompanying materials
+# are made available under the terms of the Eclipse Public License v1.0
+# which accompanies this distribution, and is available at
+# http://www.eclipse.org/legal/epl-v10.html
+#
+# Contributors:
+#     IBM Corporation - initial API and implementation
+###############################################################################
+#
+#ISMESSAGEFILE FALSE
+#NLS_ENCODING=UNICODE
+#NLS_MESSAGEFORMAT_NONE
+#
+
+description=\u3053\u306e\u30d5\u30a3\u30fc\u30c1\u30e3\u30fc\u306f\u3001Java API for XML-Based Web \u30b5\u30fc\u30d3\u30b9 2.2 \u306e\u30b5\u30dd\u30fc\u30c8\u3092\u53ef\u80fd\u306b\u3057\u307e\u3059\u3002 \
```

Comment:
wrong version

Reply:
with pre-translated files, you can't assume that the translators will spot errors or update copyright dates